### PR TITLE
Rhonda Wilhelmson Mod 4 Capstone

### PR DESCRIPTION
**Pull request by rwilhel**

### CODE REVIEW

File: 1703-b/4module/capstone_projects/proposals/rhonda.md [hunk 1]
```diff
@@ -4,24 +4,24 @@
 
 ### Pitch
 
-1 sentence that explains the value proposition of the application. How would you explain it to a potential business partner, team member, or investor?
+This application provides a user with the option to select several cryptocurrencies in order to track their real time rates with an Amazon Alexa skill extension.
```
Comment (case-eee) [inline 1703-b/4module/capstone_projects/proposals/rhonda.md]: cool!

File: 1703-b/4module/capstone_projects/proposals/rhonda.md [hunk 2]
```diff
@@ -4,24 +4,24 @@
...
 
 ### Target Audience
 
-1-3 sentences describing what type of user your app would be applicable to.
+Anyone who's curious about blockchain currencies. It will be accessible to users from all backgrounds.
 
 ### New Techniques
 
-Which new techniques are you hoping to implement?
+I am hoping to build this application with Vue.js and implement an Amazon Alexa skill to accompany it.
```
Comment (case-eee) [inline 1703-b/4module/capstone_projects/proposals/rhonda.md]: this sounds great! what API will you use to pull cryptocurrency data?

File: 1703-b/4module/capstone_projects/proposals/rhonda.md [hunk 3]
```diff
@@ -4,24 +4,24 @@
...
 
 ### Workflow
 
-What project management tool will you be using to manage your progress?
+I will be using Pivotal Tracker to manage the progress of my project.
```
Comment (case-eee) [inline 1703-b/4module/capstone_projects/proposals/rhonda.md]: 👍

File: 1703-b/4module/capstone_projects/proposals/rhonda.md (hunk 2 above)
Comment (rwilhel) [reply 1703-b/4module/capstone_projects/proposals/rhonda.md]: I was going to use ShapeShift's API because they're a local company.

Comment (case-eee) [reply 1703-b/4module/capstone_projects/proposals/rhonda.md]: awesome 👍

**Pull request merged**